chaoskube/chaoskube.go

```diff
@@ -123,8 +123,10 @@ func New(client kubernetes.Interface, labels, annotations, namespaces, namespace
 
 // Run continuously picks and terminates a victim pod at a given interval
 // described by channel next. It returns when the given context is canceled.
-func (c *Chaoskube) Run(ctx context.Context, next <-chan time.Time) {
+func (c *Chaoskube) Run(ctx context.Context, maxJitter time.Duration, next <-chan time.Time) {
```

Comment:
It would be nice if we only had one "tick source". A Ticker with a channel and some jitter inside would be nice Something like https://github.com/lthibault/jitterbug maybe.